portal-api/src/createApp.js

```diff
@@ -35,7 +36,11 @@ app.use(cors({
 }));
 
 app.use('/v1', openRouter);
+
+app.use(passport.authenticate('jwt', { session: false }));
```

Comment:
I've moved this and the cleanXss steps into this file, so that the routes file only contains the routes.

I think it's easier to follow the order that things are applied this way.

Reply:
I'm happy to leave it this way but I find it slightly more intuitive to have the 3 separate routers with different levels of protection/middleware defined on the separate routers rather than looking at the ordering of the different routers but I suspect that's personal preference!

Reply:
Hmm yes I see what you mean. I'll move them back in